reflect-lint/src/main/java/dagger/reflect/lint/IssueRegistry.java

```diff
@@ -0,0 +1,16 @@
+package dagger.reflect.lint;
+
+import com.android.tools.lint.detector.api.Issue;
+import java.util.Arrays;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public class IssueRegistry extends com.android.tools.lint.client.api.IssueRegistry {
```

Comment:
change the name so we can static import?

Reply:
Do you have a suggestion for the name? `DaggerReflectIssueRegistry`?